pkg/get/get.go

```diff
@@ -52,6 +52,11 @@ type Tool struct {
 	NoExtension bool
 }
 
+type ToolLocal struct {
```

Comment:
What purpose does ToolLocal serve? Can you add a comment to the struct to explain its role?

Reply:
This is for passing struct to template execute, we need `Path`for the templates which is not available on `Tools` struct,so created a new one to pass the name of the tool and the path to the binary after download. I guess there must be a better way to do it. @alexellis 
https://github.com/yankeexe/arkade/blob/f2dee17d17d6b435ee08b8d3fafaad2579c60bde/pkg/get/get.go#L355